### DISCUSSION TITLE
ArgoCD UI stuck with Loading when loging with OIDC

### DISCUSSION THREAD

**Discussion by tiwarisanjay** [argoproj/argo-cd#11242] [Q&A]
Hello, I am using ArgoCD OIDC connection for SSO integration. I am getting some intermittent issue on UI where it get stuck with Loading. I am using v2.4.14 argocd version. I have to restart the argocd-server pods in order to make it work again.
FYI : Using Istio sidecar in GKE. 

[ArgoCD Loading issue image](https://files.slack.com/files-pri/T08PSQ7BQ-F04ALS3DY01/image.png)

**Comment by cychoi34** [comment]
Hi, same issue here, did you get solved?

**Reply by tiwarisanjay** [reply]
I removed istio sidecar for ArgoCD and now it's fine. But couldn't solve it with istio
